spec/features/finance/npq/course_payment_breakdown_spec.rb

```diff
@@ -127,6 +136,15 @@ def then_i_should_have_the_correct_payment_breakdown_per_npq_lead_provider
     end
   end
 
+  def then_i_should_see_the_courses_vat_and_total_payment
+    within("[data-test='npq-courses-total-paid']") do
```

Comment:
*Really* not a not of fan of test code polluting the production code. This should be accessible with simple css selectors.

Reply:
It was necessary as I was getting ambiguous match with another table